docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  mongo:
+    container_name: mongo-on-docker
+    image: mongo:5.0.6
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: "${MONGO_INITDB_ROOT_USERNAME}"
```

Comment:
`${EXAMPLE}` is the notation for reading from the environment file. We have a variable in our .env file called `MONGO_INITDB_ROOT_USERNAME` and this is used to create the database, but we don't want these variables to be public for security reasons.